examples/tests/sdk_tests/util/exec.test.w

```diff
@@ -0,0 +1,64 @@
+bring util;
+bring expect;
+bring fs;
+
+test "exec() with valid program" {
+  let program = "echo";
+  let args = ["-n", "Hello, Wing!"];
+
+  let output = util.exec(program, args);
+  
+  expect.equal(output.stdout, "Hello, Wing!");
+  expect.equal(output.stderr, "");
+  expect.equal(output.status, 0);
+}
+
+test "exec() with invalid program" {
+  let program = "no-such-program";
+  let args = [""];
+
+  let output = util.exec(program, args);
+
+  expect.equal(output.stdout, "");
+  expect.equal(output.stderr, "");
+  expect.equal(output.status, "ENOENT");
```

Comment:
This looks like a bug -- the `status` should be a number, not a string. It's odd that this is compiling

Reply:
I thought the same, but that's what is actually being returned as exit status.

Reply:
I have to say that I think that a non-zero exit code and an invalid program are not the same situation. I would argue that for `exec()`, the non-zero exit code should be a "success" (e.g. return the exit code in `status`) and ENOENT should be an exception.

Reply:
I see, it sounds like you're saying that the host machine not being able to find the program (like `no-such-program`) is a different error case than a program running and returning a non-zero exit code. Throwing an exception for the first situation makes sense to me. 👍

@garysassano perhaps we can add special handling that checks if the error code is "ENOENT" and throws in that one case?

Reply:
@Chriscbr Should be ready to 🚢 now.